workflows/genome-assembly/assembly-with-flye/Genome-assembly-with-Flye-tests.yml

```diff
@@ -0,0 +1,40 @@
+- doc: Test outline for Genome-assembly-with-Flye
+  job:
+    Input sequence reads:
+      class: File
+      path: test-data/Input sequence reads.fastqsanger.gz
+      filetype: fastqsanger.gz
+  outputs:
+    Bar chart showing contig sizes:
+      path: test-data/Bar chart showing contig sizes.png
+      compare: sim_size
+      delta_frac: 0.2
+    Flye assembly on input dataset(s) (consensus):
+      path: test-data/Flye assembly on input dataset(s) (consensus).fasta
+      compare: sim_size
+      delta_frac: 0.2
+    Flye assembly on input dataset(s) (assembly_graph):
+      path: test-data/Flye assembly on input dataset(s) (assembly_graph).graph_dot
+      compare: sim_size
+      delta_frac: 0.2    
+    Flye assembly on input dataset(s) (Graphical Fragment Assembly):
+      path: test-data/Flye assembly on input dataset(s) (Graphical Fragment Assembly).txt
+      compare: sim_size
+      delta_frac: 0.2    
+    Flye assembly on input dataset(s) (assembly_info):
+      path: test-data/Flye assembly on input dataset(s) (assembly_info).tabular
```

Comment:
I think it's better to quote those path or avoid whitespaces

Reply:
will fix, thanks